terraform/terragrunt.hcl

```diff
@@ -57,10 +57,10 @@ inputs = {
     },
   }
   gcs_buckets_names = {
-    pipeline_root_bucket      = get_env("PIPELINE_ROOT_BUCKET_NAME")
+    pipeline_root_bucket      = "${local.project_id}-pipeline-root-bucket"
```

Comment:
Why are we switching from get_env? Means we have bucket names in two places. Same fir assets bucket below